<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page1_fzsz</class>
 <widget class="QWidget" name="Page1_fzsz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QTabWidget::pane {

        border-radius:5px;

}

QTabWidget::tab-bar {
        alignment: left;

}

QTabBar::tab {
        background:  rgba(192, 192, 192, 150);
        border: none;
        border-top-left-radius: 4px;
        border-top-right-radius: 4px;
        min-width: 80px;
        padding: 8px;
color:white;
}

QTabBar::tab:hover {
        color:#B0E0E6;
		background:white;

}

QTabBar::tab:selected {

		color:#33A1C9;
		background:white;

}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">

QLineEdit {
          background: rgba(255, 255, 255, 150);
          border: 1px solid #cccccc;
           border-radius: 6px;
        }
        QLineEdit:hover {
           background: rgba(255, 255, 255, 200);
        }
        QLineEdit:focus {
           border: 2px solid #66afe9;
        }
       QLineEdit:pressed {
          background: rgba(255, 255, 255, 255);
        }
</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>手动加载</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="styleSheet">
          <string notr="true">*{
font: 75 14pt &quot;Calibri&quot;;
}</string>
         </property>
         <property name="title">
          <string>手动预设加载</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="styleSheet">
             <string notr="true">QLineEdit,QSpinBox,QDoubleSpinBox{
width: 150px;
height: 40px;

}
QPushButton{
width:150px;
height:50px;
color: rgb(0, 0, 0);
}</string>
            </property>
            <property name="title">
             <string>当前负载</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>负载:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_2">
                   <property name="text">
                    <string>0</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>%</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_3">
                   <property name="text">
                    <string>0</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Kw</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider">
                 <property name="mouseTracking">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="sliderPosition">
                  <number>0</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>功率因数:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit">
                   <property name="text">
                    <string>0</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider_2">
                 <property name="maximum">
                  <number>10</number>
                 </property>
                 <property name="pageStep">
                  <number>1</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>持续时间(s):</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_4">
                   <property name="text">
                    <string>1</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>添加</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>保存</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <weight>9</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>手动预设负载</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QTableView" name="tableView">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideNone</enum>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="groupBox_13">
               <property name="styleSheet">
                <string notr="true">QLineEdit,QSpinBox,QDoubleSpinBox{
width: 150px;
height: 40px;

}
QPushButton{
width:150px;
height:50px;
 background-color: #004080;
  color: white;
  border: none;
 border-radius: 10px;
}
QPushButton:hover {
	background-color: #0080FF;
}
QPushButton:pressed {
	background-color: #0000FF;
}</string>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QPushButton" name="pushButton">
                    <property name="text">
                     <string>选择</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_12">
                    <property name="text">
                     <string>删除</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_14">
                    <property name="text">
                     <string>上升</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_15">
                    <property name="text">
                     <string>下降</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_2">
                    <property name="text">
                     <string>加载</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>自动加载</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_11">
         <property name="styleSheet">
          <string notr="true">*{
font: 75 14pt &quot;Calibri&quot;;
}</string>
         </property>
         <property name="title">
          <string>自动加载</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QGroupBox" name="groupBox_16">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>预设负载</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QTableView" name="tableView_3">
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_12">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <weight>9</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QPushButton{
width:150px;
height:50px;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
}</string>
            </property>
            <property name="title">
             <string>加载负载</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QGroupBox" name="groupBox_4">
               <property name="styleSheet">
                <string notr="true">
QPushButton{
width:150px;
height:50px;
 background-color: #004080;
  color: white;
  border: none;
 border-radius: 10px;
}
QPushButton:hover {
	background-color: #0080FF;
}
QPushButton:pressed {
	background-color: #0000FF;
}</string>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QPushButton" name="pushButton_16">
                    <property name="text">
                     <string> 选择</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_13">
                    <property name="text">
                     <string> 添加</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_17">
                    <property name="text">
                     <string>清除</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_18">
                    <property name="text">
                     <string>上升</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_19">
                    <property name="text">
                     <string>下降</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QTableView" name="tableView_2">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>150</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
